src/client/components/string-filter-menu/string-filter-menu.tsx

```diff
@@ -154,7 +154,7 @@ export class StringFilterMenu extends React.Component<StringFilterMenuProps, Str
     let menuCont: JSX.Element = null;
 
     if (filterMode === FilterMode.REGEX || filterMode === FilterMode.CONTAINS) {
-      menuSize = Stage.fromSize(350, 410);
+      menuSize = Stage.fromSize(600, 410);
```

Comment:
Why do they use a different size here? Doesn't make a lot of sense to me (especially as the smaller one is displaying a checkbox before the list entry). Lets make them the same size & extract a const for the size please